hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_order_data_manager.py

```diff
@@ -0,0 +1,284 @@
+import asyncio
+import time
+from asyncio import Task
+from typing import Dict, List, Optional
+
+from bxsolana.provider import Provider
+from bxsolana_trader_proto.api import GetOrderbookResponse, GetOrderStatusResponse, OrderbookItem, OrderStatus, Side
+
+from hummingbot.client.hummingbot_application import HummingbotApplication
+from hummingbot.connector.exchange.bloxroute_openbook.bloxroute_openbook_constants import OPENBOOK_PROJECT
+
+
+class Orderbook:
+    asks: List[OrderbookItem]
+    bids: List[OrderbookItem]
+
+    def __init__(self, asks: List[OrderbookItem], bids: List[OrderbookItem]):
+        self.asks = asks
+        self.bids = bids
+
+
+class OrderbookInfo:
+    best_ask_price: float = 0
+    best_ask_size: float = 0
+    best_bid_price: float = 0
+    best_bid_size: float = 0
+    latest_order_book: Orderbook
+
+    def __init__(
+        self,
+        best_ask_price: float,
+        best_ask_size: float,
+        best_bid_price: float,
+        best_bid_size: float,
+        latest_order_book: Orderbook,
+        timestamp: float
+    ):
+        self.best_ask_price = best_ask_price
+        self.best_ask_size = best_ask_size
+        self.best_bid_price = best_bid_price
+        self.best_bid_size = best_bid_size
+        self.latest_order_book = latest_order_book
+        self.timestamp = timestamp
+
+
+class OrderStatusInfo:
+    order_status: OrderStatus
+    quantity_released: float
+    quantity_remaining: float
+    side: Side
+    fill_price: float
+    order_price: float
+    client_order_i_d: int
+    timestamp: float
+
+    def __init__(
+        self,
+        order_status: OrderStatus,
+        quantity_released: float,
+        quantity_remaining: float,
+        side: Side,
+        fill_price: float,
+        order_price: float,
+        client_order_i_d: int,
+        timestamp: float,
+    ):
+        self.order_status = order_status
+        self.quantity_released = quantity_released
+        self.quantity_remaining = quantity_remaining
+        self.side = side
+        self.fill_price = fill_price
+        self.order_price = order_price
+        self.client_order_i_d = client_order_i_d
+        self.timestamp = timestamp
+
+    def __eq__(self, other: "OrderStatusInfo"):
+        return (
+            self.order_status == other.order_status
+            and self.quantity_released == other.quantity_released
+            and self.quantity_remaining == other.quantity_remaining
+            and self.side == other.side
+            and self.fill_price == other.fill_price
+            and self.order_price == other.order_price
+            and self.client_order_i_d == other.client_order_i_d
+            and self.timestamp == other.timestamp
+        )
+
+
+class BloxrouteOpenbookOrderDataManager:
+    def __init__(self, provider: Provider, trading_pairs: List[str], owner_address: str, start_timeout: int = 10):
```

Comment:
give a unit of time for `start_timeout`, e.g. `start_timeout_s`

Reply:
are you sure these field is necessary? I'd advise against it.

Reply:
Which fields are you referring to?

Reply:
`start_timeout`. `await`ing the initialization functions seems to already be good enough. Why force an additional 10s wait? They should begin producing updates instantly.